libraries/AP_AHRS/AP_AHRS_NavEKF.cpp

```diff
@@ -606,6 +667,12 @@ bool AP_AHRS_NavEKF::use_compass(void)
     case EKFType::SITL:
         return true;
 #endif
+
+#if HAL_EXTERNAL_AHRS_ENABLED
+    case EKFType::EXTERNAL:
+        // fall through
```

Comment:
we can't tell yet if the compass is actually being used or not. The default is to use GPS for yaw with the VN-300. I'm waiting on info on the status bits in the protocol which would allow me to fill this in